spec/datasets_cli_spec.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+require "spec_helper"
+require "datasets_cli"
+require "fileutils"
+require "find"
+
+module Datasets
+  RSpec.describe CLI, integration: true do
+    def run_cli
+      described_class.start(["all"])
+    end
+
+    INTEGRATION_HOME = File.join(File.dirname(__FILE__),"integration")
+    DATASET_ROOT = File.join(INTEGRATION_HOME,"datasets")
+    REPO_ROOT = File.join(INTEGRATION_HOME,"repo")
+    DATASETS = [%w(ht_text_pd
+                  ht_text_pd_open_access
+                  ht_text_pd_world
+                  ht_text_pd_world_open_access)].freeze
+
+    OLD_TIMESTAMP=Time.new('1970-01-01T00:00:00Z');
+    NEW_TIMESTAMP=Time.new('2017-04-01T00:00:00Z');
+
+    INTEGRATION_DB_PATH=File.join(INTEGRATION_HOME,"demo.db")
+
+    # should fully reset and re-seed for each test?
+    def reset_database(db)
+      db.run("delete from feed_audit")
+      db.run("delete from rights_current")
+    end
+
+    def reset_repository_timestamps(repo_path)
+      pairtree_prefix = File.join(repo_path,"obj/test/pairtree_root/00")
+
+      old_vol_file = File.join(pairtree_prefix,"1","001","001.zip")
+      FileUtils.touch(old_vol_file,mtime: OLD_TIMESTAMP)
+
+      new_vol_file = File.join(pairtree_prefix,"2","002","002.zip")
+      FileUtils.touch(new_vol_file,mtime: NEW_TIMESTAMP)
+    end
+
+    def add_test_data(db)
```

Comment:
This is fine and works.  My opinion is that this db should be a fixture or factory.  Given the simplicity and static nature of the project, I prefer fixture.